### ISSUE TITLE
Wallet - Collectibles view with two foldable sections

### ISSUE THREAD

**Issue by micieslak** [status-im/status-desktop#13194]
## Description

View of collectibles divide into two foldable sections with behavior analogous to assets view where regular list view is used.

![image](https://github.com/status-im/status-desktop/assets/20650004/ec6a3bf7-3ad4-4e4b-857e-1c7383e66e5e)

Depends on: https://github.com/status-im/status-desktop/issues/13193

**Comment by alexandraB99** [comment]
Duplicate of https://github.com/status-im/status-desktop/issues/13045
